README.md

```diff
@@ -24,30 +24,32 @@ steps:
     uses: actions/setup-java@v4
     with:
       distribution: 'temurin'
-      java-version: '17'
+      java-version: 17
   
   # Install the Firebase Emulator Suite
   - name: Start Firebase Emulator Suite
     uses: firebase-emulator-action@v1
```

Comment:
@Ehesp any thoughts on this? Would it be best to tag newer releases with the `ref` as the full version version `v1.0.0` resulting into `invertase/firebase-emulator-action@v1.0.0` or to shortening it to maybe `v1` resulting into `invertase/firebase-emulator-action@v1`

Reply:
In my experience publishing actions you must have the author yes, but you do *not* want to specify the full version because then people can't get semver-compatible bug fixes

You want people to use v1, and you want to have a v1 tag plus the vX.Y.Z releases, but every time you do a release you "slide" the v1 tag up to the new release so that people get the new release without effort. That's how everyone does it AFAIK

Reply:
Thank you @mikehardy for the insight. 

I truly agree thats the same developer experience i have always had using actions where just v1 is defined rather than the full latest version tag. 

Just to be sure, is this the point when we need to add a `.github/workflows/release.yaml` or `.github/workflows/release-tag.yaml`  file to achieve what you have described so as to have `vX` pointing to the full version's (eg `vX.Y.Z`) as per the latest release?

Or there is a different way to achieve it ?

Tried releasing an action and when i made patches, reusing the same `vX` tag for subsequent published patches was rejected as the tags needed to be unique.

Reply:
@mikehardy I have opened a PR #6 . Kindly have a look at it.